frontend/src/pages/create_campaign/index.tsx

```diff
@@ -31,7 +31,9 @@ const CreateCampaign = () => {
   const [tab, setTab] = useState(0);
   const [campaignName, setCampaignName] = useState("");
   const [startDate, setStartDate] = useState(new Date());
-  const [endDate, setEndDate] = useState(new Date());
+  const [endDate, setEndDate] = useState(
+    new Date(Date.now() + 14 * 3600 * 1000 * 24)
+  );
```

Comment:
oh something we could do here is use [lazy initial state](https://reactjs.org/docs/hooks-reference.html#lazy-initial-state) by wrapping this in a function so it only gets evaluated on the first render. not sure how expensive computing this date actually is but it can't hurt optimising it